README.md

```diff
@@ -44,6 +44,9 @@ cargo install --git https://github.com/dicej/wasm-tools --branch wasm-compose-re
 # Build the middleware
 cargo component build --manifest-path github-oauth/Cargo.toml --release
 
+# Build example
+(cd example && spin build)
```

Comment:
we are building the example the in spin line below with the `--build` flag. Do you still feel this line is necessary?